projects/netgrif-components/src/lib/panel/panel.module.ts

```diff
@@ -24,6 +25,7 @@ import {ImmediateFilterTextContentComponent} from './immediate/immediate-filter-
         PublicWorkflowPanelComponent,
         ImmediateFilterTextComponent,
         ImmediateFilterTextContentComponent,
+        PanelItemComponent,
```

Comment:
it might be a good idea to export the component as well to make creation of custom panels simpler

Reply:
It is exported in public.api.ts. Is ok for you?

Reply:
both is neccessary. If you export it in `public.api` then you can import the class, but if you want to use the component in your html templates it has to be exported in the module as well